docs/troubleshooting.md

```diff
@@ -94,6 +94,18 @@ sudo apt install python3-venv python3-pip
 Reference:
 [Python Packaging User Guide: Installing pip/setuptools/wheel with Linux Package Managers](https://packaging.python.org/guides/installing-using-linux-tools)
 
+## macOS issues
+
+If you want to use a Pipx installed package in a shebang (a common example is the AWS cli),
+you will likely not be able to, because the binary will be under `~/Library/Application Support/pipx`.
+The space in the path is not supported in a shebang. A simple solution is symlinking
+`~/Library/Application Support/pipx` to `~/Library/ApplicationSupport/pipx`, and using that as the
+path in the shebang instead.
+
+```
+ln -s $HOME/Library/Application Support/pipx $HOME/Library/ApplicationSupport/pipx
```

Comment:
```suggestion
mkdir $HOME/Library/ApplicationSupport
ln -s $HOME/Library/Application\ Support/pipx $HOME/Library/ApplicationSupport/pipx
```

Reply:
The original command doesn't work properly.

Reply:
Oh damn, I should have tried it in a mac VM. Thanks for verifying!